### ISSUE TITLE
Ablation question

### ISSUE THREAD

**Issue by mtchiu2** [IDEACVR/MaskDINO#10]
Thank you for your work.

In table 13, for 12-epoch settings, it is mentioned that -DINO mask branch reaches 49.5, and was 49.6 before training for segmentation. But 49.6 in Table 2 is trained for 36 epochs instead of 12? Could you clarify the setting for the -DINO mask branch setting?

**Comment by FengLi-ust** [comment]
Yes, we trian DINO for 36epochs and then finetune the mask branch for 12 epochs. It is a very strong baseline to show our performance improvement.

**Comment by mtchiu2** [comment]
Thanks for the clarification!
